File: Python3/BubbleSort.py
i=0
j=0
data = [54,78,12,2,89,25,78,29,69,35]

print(data)

for i in range(0,len(data)-1):
    for j in range(0,(len(data)-i)-1):
        if(data[j]>data[j+1]):
            tmp = data[j]
            data[j] = data[j+1]
            data[j+1] = tmp

print(data)

File: Ruby/BubbleSort.rb
def bubble_sort(ary)
	length = ary.length
	for i in 0..(length-1)
		for j in 0..((length-2)-i)
			l = j
			r = j+1
			if ary[l] > ary[r]
				tmp = ary[l]
				ary[l] = ary[r]
				ary[r] = tmp
			end
		end
	end
	return ary
end

data = [54,78,12,2,89,25,78,29,69,35]
p data
p bubble_sort(data)
